### PR TITLE
PHP 8 compatibility fixes

### PR DESCRIPTION
**Pull request by nelsonec87**
## Proposed changes:

These are minor changes as part of our PHP 8.1 compatibility update initiative pdqkMK-UN-p2.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Jetpack product discussion

* pdqkMK-UN-p2
* #23695 
## Testing instructions:

These are minor enough that I think visual inspection should be sufficient but please let me know if you need more information.

### CODE REVIEW

File: projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php [hunk 1]
```diff
@@ -444,7 +444,7 @@ public function override_svg_icons() {
 			// phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
 			$menu[ $idx ] = $menu_item;
 		}
-		if ( count( $svg_items ) > 0 ) {
+		if ( $svg_items !== [] ) {
```
Comment (markbiek) [inline projects/plugins/jetpack/modules/masterbar/admin-menu/class-base-admin-menu.php]:
```suggestion
		if ( array() !== $svg_items ) {
```

**Comment by markbiek** [comment]
@nelsonec87 This is failing a check on `array_key_last`. Does it need rebasing to get the latest phpcs rules?

**Comment by nelsonec87** [comment]
> @nelsonec87 This is failing a check on `array_key_last`. Does it need rebasing to get the latest phpcs rules?

Rebased!

**Comment by nelsonec87** [comment]
Rebased as check was failing

**Review by markbiek — APPROVED** [review]
LGTM!

**Pull request merged**